content/en/serverless/guide/serverless_tracing_and_bundlers.md

```diff
@@ -126,6 +126,78 @@ Datadog's tracing libraries (`dd-trace`) are known to be not compatible with bun
     module.exports = [nodeExternalsPlugin()]
     ```
 
+## AWS CDK & esbuild
+
+The `NodeJsFunction` construct in the AWS CDK uses esbuild. The default configuration is not compatible with Datadog's tracing libraries. The CDK allows you to override the default configuration and provide a custom esbuild file to support bundling and the Datadog tracing libraries:
+
+1. Follow the installation instructions for Node.js and ensure the Datadog Lambda layer for Node.js is added to your Lambda function.
```

Comment:
Would it make sense to also cover users who are packages instead of the layer?

Reply:
Just to confirm, you mean people who are bundling the dd-trace package inside their code and not adding the layer at all?

Reply:
So the Layer contains both `datadog-lambda-js` and `dd-trace` package. Some customers might not add layers, but instead of the package, and do the instrumentation in a different way.

At the end of the day, they can pick any of the ones they want!